sympy/parsing/tests/test_c_parser.py

```diff
@@ -880,6 +880,8 @@ def test_function_call():
 
 
     def test_parse():
+        import tempfile
```

Comment:
I think it is better to import at the top of the module

Reply:
I'm not sure about that.
The module is only used in this function and nowhere else. So, what's the point of having an import unless this function is called. That just increases the overhead.